Server/src/tests/controllers/AdminManagementController.spec.ts

```diff
@@ -129,9 +120,12 @@ describe('Fetch All Categories Materials Controller ', () => {
                 operation: 'remove'
             }
         }
-        await controller.createPermissionUpdateRequest(mockRequest as Request, mockResponse as Response)
-        expect(mockResponse.json).toBeCalledWith("No such user exists");
-        expect(mockResponse.status).toBeCalledWith(404);
+        await updatePermission(mockRequest as Request, mockResponse as Response, 404, "No such user exists")
     });
 
+    async function updatePermission(mockRequest: Request, mockResponse: Response, status: number, expectedResponse: any) {

```

Comment:
updatePermissionWithAssert (or expect) would better show what this method does, at first i saw the code and though you just removed the expect statements.

Reply:
Will add